frontend/src/app/admin/Organization/OrganizationDropDown.tsx

```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { toast } from "react-toastify";
+
+import Alert from "../../commonComponents/Alert";
+import { showNotification } from "../../utils";
+import Dropdown from "../../commonComponents/Dropdown";
+import Required from "../../commonComponents/Required";
+
+export interface OrganizationOption {
+  name: string;
+  externalId: string;
+}
+
+export interface OrganizationOptions {
+  organizations: [OrganizationOption];
```

Comment:
I am a little concerned about the size of this. I think it will work fine form now. We currently have ~350 organizations in production. 

Can you add a ticket to add an organization search?

Reply:
@nickclyde and I had discussed this a little and also agreed that this will be something that needs improvement.  I added https://github.com/CDCgov/prime-simplereport/issues/1875 to track it.